letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -264,7 +264,18 @@ UNLIKELY_EOF
       # Report error. (Otherwise, be quiet.)
       echo "Had a problem while installing Python packages."
       if [ "$VERBOSE" != 1 ]; then
+        echo
+        echo "pip prints the following errors: "
+        echo "====================================================="
         echo "$PIP_OUT"
+        echo "====================================================="
+        echo
+        echo "Certbot has problem setting up the virtual environment."
```

Comment:
Can we grep "$PIP_OUT" for the relevant error strings, so that we're only printing the hint about swap space if that's the actual error the user encountered?